.env.example

```diff
@@ -7,4 +7,4 @@ DJANGO_DATABASE_URL=postgres://mathesar:mathesar@mathesar_db:5432/mathesar_djang
 MATHESAR_DATABASES=(mathesar_tables|postgresql://mathesar:mathesar@mathesar_db:5432/mathesar)
 # Change this to Dockerfile.integ-tests to be able to run e2e integ tests
 DOCKERFILE=Dockerfile
-DJANGO_SETTINGS_MODULE=config.settings
+DJANGO_SETTINGS_MODULE=config.settings.development
```

Comment:
Shouldn't we just drop this since all relevant services have it defined with the appropriate defaults? This is just going to trip someone trying to change from dev to demo (or prod) mode, won't it?

Reply:
`.env` file is a file used by docker, we don't have control over it but  We should remove setting up a .env file in our primary setup instructions. Maybe add we can add it to our FAQ section(if we have one) incase someone wants to override the environment variables.